.ng-dev/commit-message.mts

```diff
@@ -11,10 +11,12 @@ export const commitMessage: CommitMessageConfig = {
     'multiple', // For when a commit applies to multiple components.
     'cdk-experimental/column-resize',
     'cdk-experimental/combobox',
+    'cdk-experimental/listbox',
     'cdk-experimental/popover-edit',
     'cdk-experimental/scrolling',
     'cdk-experimental/selection',
     'cdk-experimental/table-scroll-container',
+    'cdk-experimental/ui-patterns',
```

Comment:
Why don't we call this `patterns`? The `ui-` seems a bit redundant.

Reply:
@jelbourn Do you recall the reason for keeping the `ui-` prefix?

Reply:
I like the `ui-`, `patterns` by itself makes me think of like GoF design patterns or something

Reply:
Yeah, the thinking was that just "patterns" by itself was too generic

Reply:
Isn't everything we do related to building UI though? It seems a bit redundant to have it in the name. Then again, if we don't intend to make this public it might not matter 🤷‍♂️